### PR TITLE
Cargo Order Misc changes

### PR DESCRIPTION
**Pull request by Yatosamasensei**
## About The Pull Request

Moved seeds to immediately before exotic seeds in the list
Lowered prices on plasteel because they were ridiculously high.
Changed standard costume to clown costume crate, and made it not restricted	
Added a handcuff crate costs 2000 and contains 4 handcuffs

## Why It's Good For The Game

Prices for plasteel were way too high, now people might actually buy them.  Added some packs for various departments that are filling missing niches. 

## Changelog
:cl:
add: Handcuff Crate
tweak: Moved seeds to immediately before exotic seeds on the list
tweak: Changed name of standard costume crate to indicate it is for clowns.
balance: Lowered prices of plasteel
/:cl:

### CODE REVIEW

**Comment by jaw-sh** [comment]
```
code/modules/cargo/packs.dm:1040:error: /obj/item/stack/sheet/mineral/bananium/fifty: undefined type path
```

**Comment by Yatosamasensei** [comment]
> ```
> code/modules/cargo/packs.dm:1040:error: /obj/item/stack/sheet/mineral/bananium/fifty: undefined type path
> ```

Should work now

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -294,6 +294,17 @@
 					/obj/item/gun/energy/disabler)
 	crate_name = "disabler crate"
 
+/datum/supply_pack/security/disabler
```
[the remark this answers is not in the document]
Comment (Yatosamasensei) [reply code/modules/cargo/packs.dm:297]: Fixed

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -1011,17 +1022,33 @@
...
 
 /datum/supply_pack/materials/plasteel50
 	name = "50 Plasteel Sheets"
 	desc = "For when you REALLY have to reinforce something."
-	cost = 16500
+	cost = 10000
 	contains = list(/obj/item/stack/sheet/plasteel/fifty)
 	crate_name = "plasteel sheets crate"
 
+/datum/supply_pack/materials/bananium50
```
[the remark this answers is not in the document]
Comment (Yatosamasensei) [reply code/modules/cargo/packs.dm:1036]: Removed, working  on finding and changing spawn rates

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -1266,6 +1293,20 @@
 	access = ACCESS_MEDICAL
 	contains = list(/obj/machinery/iv_drip/saline)
 
+/datum/supply_pack/medical/organs
```
[the remark this answers is not in the document]
Comment (Yatosamasensei) [reply code/modules/cargo/packs.dm:1296]: Removed

File: code/modules/cargo/packs.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (Yatosamasensei) [reply code/modules/cargo/packs.dm:1036]: I'm gonna do that in a separate branch and pull request. It appears it isn't as simple as changing a number.

**Pull request closed without merging**